### PR TITLE
enh(epp): network-switchs-alcatel-omniswitch-snmp - add interfaces se…

### PR DESCRIPTION
**Pull request by garnier-quentin**
…rvice

## Target version

- [ ] 21.10.x (staging)
- [ ] 22.04.x (staging)
- [ ] 22.10.x (staging)
- [ ] Cloud (staging)
- [X] Plugin Packs (staging)
- [ ] 23.04.x (next)

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-switchs-alcatel-omniswitch-snmp.md [hunk 1]
```diff
@@ -16,31 +15,24 @@ Le Pack Centreon **Alcatel Omniswitch** apporte un modèle d'hôte :
...
+|:----------------|:--------------------------------------------|:----------------------------------------------------------|:-------|
+| Cpu             | Net-Alcatel-Omniswitch-Cpu-SNMP             | Contrôle le taux d'utilisation du CPU                     | X      |
+| Flash-Memory    | Net-Alcatel-Omniswitch-Flash-Memory-SNMP    | Contrôle le taux d'utilisation de la mémoire Flash        | X      |
+| Hardware        | Net-Alcatel-Omniswitch-Hardware-SNMP        | Contrôle l'état du hardware                               | X      |
+| Interfaces      | Net-Alcatel-Omniswitch-Interfaces-SNMP      | Contrôle les interfaces                                   |        |
+| Memory          | Net-Alcatel-Omniswitch-Memory-SNMP          | Contrôle le taux d'utilisation de la mémoire              | X      |
+| Spanning-Tree   | Net-Alcatel-Omniswitch-SpanningTree-SNMP    | Contrôle l'état du spanning tree sur les interfaces       |        |
+| Virtual-Chassis | Net-Alcatel-Omniswitch-Virtual-Chassis-SNMP | Contrôle l'état des chassis virtuels                      |        |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-switchs-alcatel-omniswitch-snmp.md:26]:
```suggestion
| Virtual-Chassis | Net-Alcatel-Omniswitch-Virtual-Chassis-SNMP | Contrôle l'état des châssis virtuels                      |        |
```

File: pp/integrations/plugin-packs/procedures/network-switchs-alcatel-omniswitch-snmp.md [hunk 2]
```diff
@@ -16,31 +15,24 @@ The Centreon Pack **Alcatel Omniswitch** brings a host template:
...
 <Tabs groupId="sync">
 <TabItem value="Service" label="Service">
 
-| Rule Name                                      | Description                                                           |
-|:-----------------------------------------------|:----------------------------------------------------------------------|
-| Net-Alcatel-Omniswitch-SNMP-Packet-Errors-Name | Discover network interfaces and monitor errored and discarded packets |
-| Net-Alcatel-Omniswitch-SNMP-Traffic-Name       | Discover network interfaces and monitor bandwidth utilization         |
+| Rule Name                                  | Description                                                             |
+|:-------------------------------------------|:------------------------------------------------------------------------|
+| Net-Alcatel-Omniswitch-SNMP-Interface-Name | Découvre les interfaces réseaux et supervise le statut et l'utilisation |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-switchs-alcatel-omniswitch-snmp.md:35]:
```suggestion
| Net-Alcatel-Omniswitch-SNMP-Interface-Name | Découvre les interfaces réseau et supervise le statut et l'utilisation |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-switchs-alcatel-omniswitch-snmp.md [hunk 3]
```diff
@@ -16,31 +15,24 @@ Le Pack Centreon **Alcatel Omniswitch** apporte un modèle d'hôte :
...
 <Tabs groupId="sync">
 <TabItem value="Service" label="Service">
 
-| Nom de la règle                                | Description                                                           |
-|:-----------------------------------------------|:----------------------------------------------------------------------|
-| Net-Alcatel-Omniswitch-SNMP-Packet-Errors-Name | Discover network interfaces and monitor errored and discarded packets |
-| Net-Alcatel-Omniswitch-SNMP-Traffic-Name       | Discover network interfaces and monitor bandwidth utilization         |
+| Nom de la règle                            | Description                                                           |
+|:-------------------------------------------|:----------------------------------------------------------------------|
+| Net-Alcatel-Omniswitch-SNMP-Interface-Name | Discover network interfaces and monitor bandwidth utilization         |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-switchs-alcatel-omniswitch-snmp.md:35]: EN dans FR et vice-versa

File: pp/integrations/plugin-packs/procedures/network-switchs-alcatel-omniswitch-snmp.md (hunk 2 above)
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-switchs-alcatel-omniswitch-snmp.md:35]: FR dans EN et vice-versa

File: pp/integrations/plugin-packs/procedures/network-switchs-alcatel-omniswitch-snmp.md [hunk 4]
```diff
@@ -16,31 +15,24 @@ The Centreon Pack **Alcatel Omniswitch** brings a host template:
...
 
 More information about discovering services automatically is available on the [dedicated page](/docs/monitoring/discovery/services-discovery)
 and in the [following chapter](/docs/monitoring/discovery/services-discovery/#discovery-rules).
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-switchs-alcatel-omniswitch-snmp.md:38]:
```suggestion
and in the [following chapter](/docs/monitoring/discovery/services-discovery#discovery-rules).
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-switchs-alcatel-omniswitch-snmp.md [hunk 5]
```diff
@@ -73,34 +65,17 @@ Could not retrieve metrics
 Could not retrieve metrics
```
[the remark this answers is not in the document]
Comment (garnier-quentin) [reply i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-switchs-alcatel-omniswitch-snmp.md:65]: i only update interface service

**Review by cg-tw — APPROVED** [review]

**Review by omercier — APPROVED** [review]
@garnier-quentin how will users compensate for lost service templates?

**Review by cg-tw — APPROVED** [review]

**Pull request merged**